packages/circuit-ui/components/ToastContext/ToastContext.module.css

```
@@ -9,13 +8,28 @@
   padding: 0 var(--cui-spacings-giga);
 }
 
+.top,
+.top-right {
+  top: var(--cui-spacings-giga);
+}
+
+.bottom {
+  bottom: var(--cui-spacings-giga);
+}
+
 @media (min-width: 480px) {
   .base {
     left: 50%;
     width: auto;
     padding: 0;
     transform: translateX(-50%);
   }
+
+  .top-right {
+    right: var(--cui-spacings-giga);
+    left: inherit;
+    transform: translateX(0%);
+  }
```

Instead of overriding the base styles here, I'd rather split the styles for each position into separate blocks:

```suggestion
.base {
    width: auto;
    padding: 0;
  }

  .top,
  .bottom {
    left: 50%;
    transform: translateX(-50%);
  }

  .top-right {
    right: var(--cui-spacings-giga);
  }
```

---

thanks, adjusted in https://github.com/sumup-oss/circuit-ui/pull/2577/commits/420600bff32c911489c6dd78b419a2825afa12a1 and [e88777e](https://github.com/sumup-oss/circuit-ui/pull/2577/commits/e88777e2088f7f4faed870d277e810129880fd45)

i kept `left: inherit` in `top-right` on desktop bc mobile styles have `left: 0` which are also necessary for `top-right`.